### ISSUE TITLE
Systemtray icon size control

### ISSUE THREAD

**Issue by BrianA-MN** [lumina-desktop/lumina#567]
New feature request if not a bug.  The systemtray icons size according to the panel size in 1.4.0.p1. However, some icons remain smaller.  I.E. SMPLAyer remains small, while BlueMan and Pidgin grow.  It would be nice if the systemtray had some additional controls for 1) how to add an item to the systemtray without it being in the Autostart menu, 2) a way to control the size of icons 3) a way if using a Vertical panel on right or left edge to have the systemtray display two columns rather than a single one with large icons if usnig more than 41 pixels.

**Comment by q5sys** [comment]
I went ahead and labeled this a bug because of 3) in your list.  I've been unable to use vertical panels myself for this very reason.  This has been something Ken and I have talked about previously, but we've never gotten around to.

**Comment by BrianA-MN** [comment]
Thanks again.  I hope this is what you were hoping to see. Honestly, I'm still confused why some of the systemtray icons are small (specifically my ESET NOD32 4 Linux and SMPlayer) but other are large, specifically WICD, volumeicon, KeePassXC, HPLIP Status Service, Blueman Applet, and Pidgin.  Running Lumina 1.4.0 p1 on Slackware64-14.2 with multilib and Nouveau driver on a NVIDIA GeForce 8400GS.

**Comment by beanpole135** [comment]
System Tray icons are small because we do not support "enlarging" them beyond the maximum size provided by the individual application (if we do that the icon start looking **really** bad and get all pixellized).

Just for background: the icons that are found for the system tray are fixed-size images provided by the application itself ([See the specification here](https://specifications.freedesktop.org/systemtray-spec/systemtray-spec-0.3.html)). Part of that spec allows for the app to provide a wide range of icon sizes, and most applications will provide both smaller and larger icons. In this case, it sounds like SMPlayer only provides a single, small icon for the system tray, so it might help to ask the upstream project to use a larger-resolution image for their system tray integration.

**Comment by BrianA-MN** [comment]
Ok, that explains why some of the icons are small.  But the other icons do grow.  So the systemtray is allowing for resizing of icons.  What I'm hoping the issue will address is the vertical panels and that systemtray icons grow to really large sizes, utilizing more space of the vertical bar than is reasonable.  Can't they be limited and put into to vertical rows?  Thanks

**Comment by q5sys** [comment]
I do have an idea to address this, but I want to talk it over with Ken before I put the time into redoing it.  With the panel changing with v2, I dont want to spend to many cycles on changing it... just for it all to be tossed out.

**Comment by BrianA-MN** [comment]
Hi Ken, 
I'm still using Lumina on my Slackware 14.2 x64.  I've always run in to
an issue though because when I press the "End" button, which I often do
to reach the end of a line in terminal, it opens a the start menu, How
do I get it to return to standard action?  "End" is not mapped in the
Keyboard shortcuts.  I can't find it mapped in any of my default
configs in /home/brian/.config/lumina-desktop/xxx.conf files either.
I'm totally stuck, where can this action be changed? 

I also use an emachines 104 qwerty keyboard, which
has special keys for launching applications, like E-mail, Search and
Internet.  I got the volume-up down to remap, but I don't see a map for
launching applications.  I tried simply seeing what the keyboard sends
by remapping a different function, but when adding that under
"advanced" it didn't take, even after a reboot of Lumina

Pax tecum, absit iniuria;
BrianA_MN
Mounds View MN USA

Addendum:
A: Because it makes understanding the conversation harder!
Q: What is wrong with top replying?
